Bottom: 074dfe414fa810a9cde4e395a65970c0b02108c4
Top:    22e3b80e1c1bc2fd08f8b277615eb4de224b84af
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-03-02 00:16:54 -0300

shift operators are not assignment operators

---

diff --git a/libcc1/libcp1plugin.cc b/libcc1/libcp1plugin.cc
index ed28443..7032ac0 100644
--- a/libcc1/libcp1plugin.cc
+++ b/libcc1/libcp1plugin.cc
@@ -602,11 +602,9 @@ plugin_new_decl (cc1_plugin::connection *self,
 	      break;
 	    case CHARS2 ('l', 's'): // operator <<
 	      opcode = LSHIFT_EXPR;
-	      assop = true;
 	      break;
 	    case CHARS2 ('r', 's'): // operator >>
 	      opcode = RSHIFT_EXPR;
-	      assop = true;
 	      break;
 	    case CHARS2 ('l', 'S'): // operator <<=
 	      opcode = LSHIFT_EXPR;
